internal/injector/injector.go

```diff
@@ -49,6 +50,8 @@ type (
 		// Dir is the working directory to use for resolving dependencies, etc... If blank, the current working directory is
 		// used.
 		Dir string
+		// IncludeTests requests test files to be prepared for injection, too.
+		IncludeTests bool
```

Comment:
I guess we _could_ decide that Orchestrion does not, ever, modify test files, but this feels somewhat arbitrary... If we don't do that, then we have to instead make it so that the injector early-returns if there's _only_ `_test.go` files in the source files list... I'm happy either way (this one way appears to be more versatile?)